sbt-cross/src/main/scala/sbtcross/Platform.scala

```diff
@@ -0,0 +1,10 @@
+package sbtcross
```

Comment:
If we decide to go for `sbtcross` as the name for the project we should probably change the name of the repo too.